torch/nn/_functions/thnn/rnnFusedPointwise.py

```diff
@@ -10,37 +10,48 @@ def __init__(self):
     def forward(self, input_gate, hidden_gate, hx, ibias=None, hbias=None):
         if self.backend is None:
             self.backend = type2backend[type(input_gate)]
+
         hy = input_gate.new()
+        storage = input_gate.new().resize_(hx.numel() * 5)
+
+        self.hasBias = False
```

Comment:
nit: can you use underscores instead of camel case? that's what we're using everywhere in Python